src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -165,6 +165,7 @@ public String getCaPassPhrase() {
      * @param getCertificateRequest get certificate request
      * @throws CertificateGenerationException if unable to generate certificate
      */
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
```

Comment:
is this needed here? There are no other changes in this file.

Reply:
My mistake, I was trying something and forgot to remove it